native/cocos/renderer/gfx-validator/DeviceValidator.cpp

```diff
@@ -24,6 +24,8 @@
 ****************************************************************************/
 
 #include "base/Log.h"
+#include "application/ApplicationManager.h"
+#include "platform/interfaces/modules/IXRInterface.h"
```

Comment:
I remember these modifications is submitted by xr team. Do I missed something?

Reply:
Updated. There was a time I checkout to a halfway PR, not clear why after then pull to latest didn't merge that latest PR in.